launcher/_patch_mc.py

```
@@ -0,0 +1,183 @@
+p = r"D:\OKApps\launcher\launcher.py"
+cfg = r"D:\OKApps\launcher\config.json"
+
+s = open(p, encoding="utf-8").read()
+c = open(cfg, encoding="utf-8").read()
+
+repls = [
+    # 1) import urllib.parse
+    ("import urllib.request\nimport urllib.error\n",
+     "import urllib.request\nimport urllib.error\nimport urllib.parse\n"),
+
+    # 2) REPOS 三元组加 rid（ok-nte/okww/ok-end-field，取自各游戏 README 官方 MirrorChyan 链接）
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)